src/main/java/com/the/dark/side/crew/fejsbuk/service/UserService.java

```diff
@@ -0,0 +1,22 @@
+package com.the.dark.side.crew.fejsbuk.service;
+
+import com.the.dark.side.crew.fejsbuk.model.User;
+import com.the.dark.side.crew.fejsbuk.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public User getUser(long id) {
```

Comment:
Here just return Optional, and handle it at controller returning 200 or 404 code